infrastructure/smart-contracts/contracts/drops/TinyKeysAirdrop.sol

```diff
@@ -162,10 +164,16 @@ contract TinyKeysAirdrop is Initializable, AccessControlUpgradeable {
                 
                 // Stake the keys
                 PoolFactory2(poolFactoryAddress).stakeKeysAdmin(poolAddress, stakeKeyIds, owner);
+                poolsProcessed++;
 
                 // Emit the event
                 emit AirdropSegmentStakeComplete(owner, poolAddress, stakeKeyIds[0], stakeKeyIds[stakeKeyIds.length-1]);
             }
+
+            if (poolsProcessed == 2) {
+                endingKeyId = startingKeyId + (i - startingKeyId);
```

Comment:
This should actually be `endingKeyId = i;`

Reply:
Lol yep you're right. Was overthinking that one 😆